templates/artist/artist_detail.html

```diff
@@ -24,23 +24,26 @@ <h1 class="header-font">{{ artist.name }}</h1>
     <div class="break">
         <hr>
     </div>
-    {% thumbnail artist.photo.img "500" as thumb %}
-        <img src="{{ thumb.url }}" width="{{ thumb.width }}" height="{{ thumb.height }}" alt="{{ artist.name }}" />
-    {% endthumbnail %}
+    <div class="extra-padding">
+        {% thumbnail artist.photo.img "500" as thumb %}
+            <img src="{{ thumb.url }}" width="{{ thumb.width }}" height="{{ thumb.height }}" alt="{{ artist.name }}" />
+        {% endthumbnail %}
+    </div>
+    <!-- Genre -->
     {% with genres=artist.genres.all %}
     <div class="break">
         <hr>
     </div>
-        <p>Genre{{ genres|length|pluralize }}: {{ genres|join:" / " }}</p>
+    <h4>Genre{{ genres|length|pluralize }}: {{ genres|join:" / " }}</h4>
     {% endwith %}
-    <p>Location: {{ artist.location }}</p>
+    <h4>Location: {{ artist.location }}</h4>
     <div class="break">
         <hr>
     </div>
     {% if campaign %}
         <!-- Goal -->
-        <h2>Goal</h2>
-        <p>${{ campaign.amount|intcomma }} {{ campaign.reason }}</p>
+        <h2>GOAL</h2>
+        <h4>${{ campaign.amount|intcomma }} for {{ campaign.reason }}</h4>
```

Comment:
I don't think we should put the word "for" here. It works if the reason is "for new EP", but what if the artist wants to write here "to shoot a music video" or "to record an EP".